File: codewars/6kyu_multiples_of_3_or_5.rb
# https://www.codewars.com/kata/514b92a657cdc65150000006

# def solution(number)
#   # put your solution here
#   sum = 0
#   while number > 0
#     number -= 1
#     if number % 3 == 0 || number % 5 == 0
#       sum += number
#     end
#   end
#   sum
# end

def solution(number)
  # put your solution here
  (1...number).select { |num| num % 3 == 0 || num % 5 == 0 }.sum
end

File: codewars/6kyu_multiples_of_3_or_5.js
// https://www.codewars.com/kata/multiples-of-3-or-5/train/javascript

function solution(number){
  if(number <= 0) { return 0 }
  let sum = 0
  
  for(i = 0; i < number; i++) {
    if(i % 3 === 0 || i % 5 === 0) {
      sum += i
    }
  }
  return sum
}